lib/datasources/nfc/nfc.dart

```diff
@@ -61,9 +62,11 @@ int a = 0;
               '\nNDEF Capacity: ${tag.ndefCapacity}'
               '\nTransceive Result:\n$result');
 
+          // format result
+          if (result.indexOf("text=") >= 0) result = result.substring(result.indexOf('text=')+5);
```

Comment:
Might be wise here to do result.startsWith incase someone has a result not starting with text but its a url with ?type=1&text=Hello

or are you purposely trying to find where text= is because sometimes its prefixed?